src/Compute/Compute/Generated/ResourceSku/ResourceSkuListMethod.cs

```diff
@@ -36,13 +36,26 @@ namespace Microsoft.Azure.Commands.Compute.Automation
     [OutputType(typeof(PSResourceSku))]
     public partial class GetAzureRmComputeResourceSku : ComputeAutomationBaseCmdlet
     {
+        [Parameter(Mandatory = false, Position = 0, ValueFromPipelineByPropertyName = true)]
```

Comment:
NIT: it would be better if it is changed to multiple lines as follows:  

[Parameter(
       Mandatory = false, 
       Position = 0, 
       ValueFromPipelineByPropertyName = true)]

Then, when each property is changed later, we can see which one is changed clearly.